app/Http/Controllers/StaticController.php

```diff
@@ -25,5 +25,9 @@ public function dashboard()
         $user = Auth::user();
         return view('dashboard',compact('user'));
     }
+    public function team()
+    {
+        return view('about-us',compact('user'));
```

Comment:
You don't have a variable named `user` here. We need the user for the dashboard links in the header navigation. So please add `$user = Auth::user();`